MTA-6448 Rules Development guide DITA modifications (#349)
migtools/mta-documentation: assemblies/rules-development-guide/assembly_logical-chaining-custom-variables.adoc

[role="_abstract"]
You can create complex conditions in rules by using the following:

> * Logical conditions inform the provider how to handle more than one condition in a when block. The analyzer provides two basic logical conditions, `and` and `or`, which you can use to aggregate results of other conditions.

Reviewer: ```suggestion
* Logical conditions inform the provider how to handle more than one condition in a `when` block. The analyzer provides `and` and `or` basic logical conditions that you can use to aggregate results of other conditions.
```
